Jaeger.sln

```diff
@@ -30,7 +31,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jaeger", "src\Jaeger\Jaeger
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "examples", "examples", "{1ADE2BBF-56C4-433D-9B1C-A532915058D3}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Jaeger.Example.WebApi", "examples\Jaeger.Example.WebApi\Jaeger.Example.WebApi.csproj", "{258F726B-6E51-4F18-9FB6-41D09A836904}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Jaeger.Example.WebApi", "examples\Jaeger.Example.WebApi\Jaeger.Example.WebApi.csproj", "{258F726B-6E51-4F18-9FB6-41D09A836904}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Jaeger.Example.WebApi.NetFx", "examples\Jaeger.Example.WebApi.NetFx\Jaeger.Example.WebApi.NetFx.csproj", "{4984D8D3-E8FE-4AC8-B285-AC041CB9B30F}"
```

Comment:
It looks like you have renamed our project to `Jaeger.Example.WebApi` and created yours as it. Please fix the GUID so that our ID stays the same.

Reply:
It looks like VS wants to update the GUID. Change reverted.